### ISSUE TITLE
Users should be able to rename devices on the sync chain

### ISSUE THREAD

**Issue by jsecretan** [brave/brave-browser#11769]
## Description 
Occasionally users would like to be able to add two similar devices to the chain, and it would be good if they could rename the devices as they arrived.

**Comment by chakrihacker** [comment]
Any update on this? I have multiple macs and can't differentiate one to another

**Comment by jsecretan** [comment]
Not yet, but I think we can bump the priority here.

**Comment by Brave-Matt** [comment]
Getting more requests for this more frequently:
https://community.brave.com/t/rename-device-in-sync/259565/3

**Comment by tortino** [comment]
yes, on android I'd like to identify the release brave from nightly one.
currently it's impossibile because the device name is the same.

**Comment by sudomakecoffee** [comment]
Renaming devices would be extremely useful, instead I have to remember my phone's model instead of a convenient name.  Hire me to work on it!

**Comment by adrianinsaval** [comment]
any news here?

**Comment by ashutoshsaboo** [comment]
Any update on this? This is a pretty essential feature to differentiate devices easily in the sync chain.

**Comment by jeffnewbold** [comment]
This feature would literally be the last thing that a browser not named Firefox would need to be able to do (re: #11769, #3298, and #13461)

**Comment by lwbaqueros** [comment]
Hi, is there any advance in this issue? It isn't very pleasant to have a very long list of devices with names I am not familiar with.

**Comment by AlexeyBarabash** [comment]
@lwbaqueros , no, currently there is no progress on this issue.

**Comment by Brave-Matt** [comment]
Another +1 
https://community.brave.com/t/please-do-feature-request-11769-rename-device-in-sync/494908/3

**Comment by cliffporterx** [comment]
2020, 2021, 2022, 2023... 2024 coming soon!
🤔

**Comment by Coffeeling** [comment]
Another request for this.

**Comment by lwbaqueros** [comment]
They gave us hope!!!

[https://community.brave.com/t/please-do-feature-request-11769-rename-device-in-sync/494908/4](https://community.brave.com/t/please-do-feature-request-11769-rename-device-in-sync/494908/4)

**Comment by victorv** [comment]
Using the latest version of Brave for OSX Ventura 13.6.2 - Brave is up to date
[Version 1.60.118 Chromium: 119.0.6045.163 (Official Build) (arm64)](https://brave.com/latest/)

And just added two phones to the sync chain and in both cases Brave failed to retrieve the actual phone name and just added a generic "iPhone" name. Ideally, it would first retrieve the proper name then allow renaming. Thank you for Brave - it's my default Browser!

**Comment by markuszeller** [comment]
Now I have 4 devices with the same name, because of multiple profiles and installations on Windows and inside WSL2 on the same device. When I want to send a link to a specific instance from another device I can not find the correct target.

**Comment by dukethorion** [comment]
It's really neat when you have multiple browsers (Stable, Beta, and Nightly), and then have the same three on a separate work profile.  I now have six (6) "Pixel 5" devices listed in my sync chain.  Not helpful in the slightest.  Either it was overlooked years ago or Brave decided that people only have one phone and one computer.

Please prioritize this issue.   Sync Settings --> Device Name

**Comment by plangin** [comment]
Switched to Brave Browser and I‘m surprised that such an obvious need is still not implemented and this issue is open for almost 4 years.

**Comment by thommesborg** [comment]
I'm irritated like @plangin that this feature isn't done by now.

**Comment by wigbam** [comment]
Why is this feature taking so long to implement? It seems like a really basic user customisable configuration option request. Are there any non-obvious internal  sysgem epercussions or something? If there are - can those be stated at least instead of just ignoring the subject completely?

**Comment by cwross** [comment]
Another vote for this feature. I'd consider the Sync feature only half-baked until this is added.

**Comment by AdnanHodzic** [comment]
Would be great to hear if there are any updates from Brave’s side regarding the 4-year-old feature request, thanks.

**Comment by LonginusKrux** [comment]
I'm having the same issue. Two identical devives and I can't rename them, neither before or after joining the chain.

Can we give this a bit of priority, please?

**Comment by Lenzelot** [comment]
With 6 devices that I recently used for my sync chain, my list of devices is now a convoluted mess, so I found this ticket. An ability to rename synced devices would make a lot of sense.

**Comment by xeruf** [comment]
I have three different iPhones on the chain - if the default names would at least use the device name or the model number instead of just "iPhone", I could at least tell them apart...

**Comment by bluedevil678** [comment]
+1 Vote for this feature, renaming on a sync chain is an important security feature as to identify. When devices share similar or default names, it becomes difficult to determine which system holds the most recent or critical data, increasing the risk of confusion, overwriting, or data loss. Unique names ensure accurate tracking, simplify troubleshooting, and make it easy to manage synchronisation priorities and data integrity across the chain.

**Comment by leaflet-frizzle0t** [comment]
+1 It would be nice if there was an option to edit device names when connecting, or if they were the same as the device name set in the iOS and iPadOS settings.

**Comment by jacobgraf** [comment]
Pleeeeease! I have like 4 copies with the same name and one is my main laptop. I don't want to delete any if I don't know which is which. 😭

**Comment by abdallh12345** [comment]
> ## Description 
> Occasionally users would like to be able to add two similar devices to the chain, and it would be good if they could rename the devices as they arrived.  
> https://finance.yahoo.co.jp/abdallh12345/998407.0

- abdallh12345 https://.yahoo.co.jpy992ZFZ6DcP9KXChgalaxy rack question about tail talent jazz keen vague napkin paddle mad unable earth quick easily half hair gallery cactus mad unable garden rack /abdallh12345 /998407.O

**Comment by abdallh12345** [comment]
> ## Description 
> Occasionally users would like to be able to add two similar devices to the chain, and it would be good if they could rename the devices as they arrived.  
> https:// docs.github.com.yahoo.co.jp/abdallh12345 /998407.O

**Comment by abcpeak** [comment]
+1

**Comment by thommesborg** [comment]
Priority P3 since 2021 @AlexeyBarabash ?

**Comment by GabrielBoehme13** [comment]
+1

Seems like a potential security problem waiting to happen when you can't properly identify sync'd devices by name.

**Comment by bsclifton** [comment]
cc: @fmarier

**Comment by Coffeeling** [comment]
Bump, definitely something I've wanted for years.

**Comment by GabrielBoehme13** [comment]
For some wider perspective on this request, I also use another privacy-focused browser which released their own device sync service a couple years ago. Right from the start, users could edit device nicknames, and in my experience it has always worked flawlessly.

With this experience setting my expectations, it feels increasingly weird that I can't edit the names of sync'd devices in Brave. I can understand that Brave's sync service likely wasn't built with this in mind originally, which may make it more difficult to add on after the fact. But we're just a few months shy of six years since this issue was first opened – are the technical challenges truly that formidable?

**Comment by abcpeak** [comment]
Now that we have AI assistance to help write code, can this problem be solved? After all, it's really confusing when two devices display the same name while syncing.

**Comment by amirrh6** [comment]
Why doesn’t Brave at least give a short reply? Do they not care at all? @bsclifton @AlexeyBarabash

**Comment by AlexeyBarabash** [comment]
@amirrh6 this is in my tasks list

**Comment by amirrh6** [comment]
@AlexeyBarabash Godspeed, Alexey. Looking forward to seeing it land in Brave.
